packages/terra-consumer-layout/CHANGELOG.md

```diff
@@ -4,3 +4,6 @@ ChangeLog
 # 0.1.0-BETA.1 - (September 06, 2017)
 -----------------
 Initial beta release for the layout component
+
+### Changed
+- Fixed the help-button alignment issue.
```

Comment:
I released a new beta today, so i think this needs to be part of a new beta tag in the changelog